backend/src/main/resources/liquibase/create-index-changelog.xml

```diff
@@ -1161,4 +1161,29 @@
             <column name="time_updated"/>
         </createIndex>
     </changeSet>
+
+    <changeSet author="anandJ" id="index_labels_mapping_entity_hash_entity_type">
```

Comment:
roll back tag missing

Reply:
move this to db-changelog-2.0.xml

Reply:
Moved to db-changelog-2.0.xml with rollback tag